src/snowflake/cli/plugins/rpc/manager.py

```diff
@@ -0,0 +1,124 @@
+from __future__ import annotations
+
+import importlib
+import inspect
+import os
+import pkgutil
+from typing import Optional, TypedDict
+
+from pygls.server import LanguageServer
+from snowflake import connector
+from snowflake.cli.api.sql_execution import SqlExecutionMixin
+from snowflake.snowpark import Session
+
+MAX_WORKERS = 5
+
+
+class ConnectionParams(TypedDict):
+    sessionToken: Optional[str]
+    masterToken: Optional[str]
+    account: Optional[str]
+    host: Optional[str]
+    port: Optional[str]
+    protocol: Optional[str]
```

Comment:
Why this definition is duplicated? We already defined it in other file

Reply:
We may have to have two. One where it's for the Dict coming in from the extension, and another where it is a dataclass used by the other layer.